docs/gettingstarted/sandbox.md

```diff
@@ -0,0 +1,129 @@
+
+
+To see logs for your stack run:
+
+ff logs demo
+```
+
+
+
+## Sandbox Layout
+
+![Sandbox Messages](../images/sandbox/sandbox_broadcast.png)
+
+The Sandbox is split up into three columns:
+
+### Left column: Prepare your request
+On the left-hand side of the page, you can fill out simple form fields to construct messages and more. Some tabs have more types of requests on them in sections that can be expanded or collapsed. Across the top of this column there are three tabs that switch between the three main sets of functionality in the Sandbox. The next three sections of this guide will walk you through each one of these.
+
+The first tab we will explore is the **MESSAGING** tab. This is where we can send broadcasts and private messages.
+
+### Middle column: Preview server code and see response
+As you type in the form on the left side of the page, you may notice that the source code in the top middle of the page updates automatically. If you were building a backend app, this is an example of code that your app could use to call the FireFly SDK. The middle column also contains the button to actually send the request.
+
+### Right column: Events received on the WebSocket
+On the right-hand side of the page you can see a stream of events being received on a WebSocket connection that the backend has open to FireFly. For example, as you make requests to send messages, you can see when the messages are asynchronously confirmed.
+
+## Messages
+
+The Messages tab is where we can send broadcast and private messages to other members and nodes in the FireFly network. Messages can be a string, any arbitrary JSON object, or a binary file. For more details, please see the tutorial on [Broadcasting data](../tutorials/broadcast_data.md) and [Privately sending data](../tutorials/private_send.md).
+
+### Things to try out
+
+- Send a broadcast message and view the data payload in every member's FireFly Explorer
+- Send a private message to one member, and verify that the data payload is not visible in the third member's FireFly Explorer
+- Send an image file and download it from another member's FireFly Explorer
+
+![Sandbox Messages](../images/sandbox/sandbox_broadcast_result.png)
+
+## Tokens
+
+The Tokens tab is where you can create token pools, and mint, burn, or transfer tokens. This works with both fungible and non-fungible tokens (NFTs). For more details, please see the tutorial on [Minting some tokens](../tutorials/mint_tokens.md).
+
+### Things to try out
+
+- Create a fungible token pool and mint some tokens and view your balance in the FireFly Explorer
+- Transfer some amount of those tokens to another member and view the transfer transaction in the FireFly Explorer
+- Burn some amount of tokens and view the transaction and updated balances in the FireFly Explorer
+- Create a non-fungible token pool and mint some NFTs
+- Transfer an NFT to another member and verify the account balances in the FireFly Explorer
+
```

Comment:
maybe also add a suggestion to create a tx failure like you showcased in the demo.  

"attempt to transfer an amount that exceeds your balance, and investigate the associated event"

Reply:
I would suggest that... except it kind of breaks the sandbox right now